regparser/web/jobs/models.py

```diff
@@ -1,16 +1,24 @@
 from django.db import models
 
+job_status_pairs = (
+    ("complete", "complete"),
+    ("complete_with_errors", "complete_with_errors"),
+    ("failed", "failed"),
+    ("in_progress", "in_progress"),
+    ("received", "received")
+)
+job_status_values = [j[0] for j in job_status_pairs]
```

Comment:
It doesn't look like `job_status_pairs` is ever used; if not, can we just make a tuple with the five strings? I'd argue a tuple is better than a list here due to being immutable.